cbopensource/connectors/taxii/bridge.py

```diff
@@ -161,35 +179,94 @@ def _import_collection(self, client, site, collection, data_set=False):
             logger.info("Feed start time %s" % feed_helper.start_date)
         logger.info("polling Collection: {}...".format(collection.name))
 
+        reports, new_feed_summary = self.get_reports_from_collection(collection, reports_limit=site.get('reports_limit'), default_score=site.get('default_score'), date_range_helper=feed_helper, data_set=data_set)
+
+        if not data_set:
+            #
+            # We only want to concatenate if we are NOT a data set, otherwise we want to refresh all the reports
+            #
+            logger.info("Adding existing reports...")
+            reports = feed_helper.load_existing_feed_data() + reports
+
+        logger.info("Total number of reports: {}".format(len(reports)))
+
+        if site.get('reports_limit') < len(reports):
+            logger.info("Truncating reports to length {0}".format(site.get('reports_limit')))
+            reports = reports[:site.get('reports_limit')]
+
+        data = build_feed_data(sanitized_feed_name,
+                               "%s %s" % (site.get('site'), collection_name),
+                               feed_summary if not new_feed_summary else new_feed_summary,
+                               site.get('site'),
+                               site.get('icon_link'),
+                               reports)
+
+        if feed_helper.write_feed(data):
+            feed_helper.save_details()
+
         #
-        # Build up the URI for polling
+        # Create Cb Response Feed if necessary
         #
 
-        if not site.get('poll_path', ''):
-            uri = None
-        else:
-            uri = ''
-            if site.get('use_https'):
-                uri += 'https://'
+        feed_id = None
```

Comment:
Oh I see, you just moved stuff around....  It's easier to do that by extracting code into its own function.